titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/JobManagementResource.java

```diff
@@ -111,6 +113,18 @@ public Response setInstances(@PathParam("jobId") String jobId,
         return Responses.fromCompletable(jobServiceGateway.updateJobCapacity(jobCapacityUpdate));
     }
 
+    @PUT
+    @ApiOperation("Update an existing job's capacity. Optional attributes min / max / desired are supported.")
+    @Path("/jobs/{jobId}/capacityAttributes")
+    public Response setCapacityWithOptionalAttributes(@PathParam("jobId") String jobId,
```

Comment:
Do we want a deprecation message or anything on the `setInstance()` call?

Reply:
Yeah I think we should deprecate it, although the deprecation marker should go in the protobuf definition